docs/hermes.md

```diff
@@ -52,3 +52,25 @@ $ react-native run-android --variant release
 ```
 
 This will compile JavaScript to bytecode during build time which will improve your app's startup speed on device.
+
+## Debugging Hermes using the Chrome Inspector
+
+Hermes supports the Google Chrome Inspector debugging protocol. This means Chrome's debug tools can be used to directly debug JavaScript running on Hermes, on an emulator or device.
+
+For Chrome to connect to a device you must have a Metro server running and connected to your target device. A Metro server can be explicitly started by running `yarn start` in the root of your app's source repository, or run implicitly for the duration of the `react-native run-android` process.
```

Comment:
You can assume that people know how to run Metro, the second sentence is superfluous.